.circleci/config.yml

```diff
@@ -162,11 +157,7 @@ jobs:
   release:
     <<: *container_config
     steps:
-      - *restore_repo
-      - *sum_lockfiles
-      - *restore_npm_cache
-      - *npm_install
-      - *github_authenticity
```

Comment:
why did github_authenticity get removed on this job?

Reply:
Yeah, it's not part of the other setup steps and I don't think I should've removed it.  Thanks for that!